eng/scripts/smoketest/Get-Package-Version.ps1

```diff
@@ -23,7 +23,8 @@ function Get-SmokeTestPkgProperties
 
     $NugetSource="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v2"
     $allPackages = Get-AllPkgProperties
-    $newPackages = $allPackages.Where({ $_.IsNewSdk })
+    $excludePackages = (Get-Content "$PSScriptRoot/SmokeTestExcludePackage.json" | ConvertFrom-Json).ExcludePackages
```

Comment:
I think you need `-Raw` here, otherwise `Get-Content` will return a newline delimited array instead of a json string (see [docs](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-content?view=powershell-7.2#example-6-get-raw-content))

Reply:
Given this list is only for this one smoke test project I'd suggest just doing it inline in the PS script similar to https://github.com/Azure/azure-sdk-for-js/blob/main/eng/scripts/Language-Settings.ps1#L235-L238 and include a link to the issue as well.